tests/test_subtraction.py

```diff
@@ -0,0 +1,10 @@
+from staff import pxl
+
+
+def test_subtraction():
```

Comment:
I'd better have tests where nominal in result, tests when in result each component could be lower 0 (3 separate tests), might be tests for zero-case (not only for subtraction).